### DISCUSSION TITLE
[@types/webextension-polyfill] Missing field definition on browser.Runtime.Port

### DISCUSSION THREAD

**Discussion by vozeldr** [DefinitelyTyped/DefinitelyTyped#56707] [Issues with a @types package]
According to MDN, there is an "optional" error property added to the port if it was disconnected due to an error. It's missing from `runtime.d.ts`

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/Port

[a comment by an automated account is not shown]

**Comment by Lusito** [comment]
Thanks for the report. I will fix this, but just so you know: The documentation says:
> Note that in Google Chrome port.error is not supported: instead, use runtime.lastError to get the error message.

So maybe it would be best if you used that instead.

**Comment by Lusito** [comment]
fixed in the next version: #56740
